src/Toml/Bi/Map.hs

```diff
@@ -572,6 +578,26 @@ _IntSet :: TomlBiMap IS.IntSet AnyValue
 _IntSet = iso IS.toList IS.fromList >>> _Array _Int
 {-# INLINE _IntSet #-}
 
+_KeyText :: TomlBiMap Key Text
+_KeyText = BiMap
+    { forward = Right . keyToText
+    , backward = textToKey
+    }
+
+_KeyString :: TomlBiMap Key String
+_KeyString = BiMap
+    { forward = Right . T.unpack . keyToText
+    , backward = textToKey . T.pack
+    }
```

Comment:
Can you add some documentation to these functions and `@since` annotation? I'm not sure at the moment, what the next version will be, but having some `@since` annotation will save some time in future. Something like this should work for now:

```haskell
{- | Bidirectional converter between 'Key' and 'Text'.

@since x.x.x.x 
-}
_KeyText :: ....
```